sdk/resourcemanager/Azure.ResourceManager/src/Common/Custom/Models/WritableResourceData.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using Azure.Core;
+
+namespace Azure.ResourceManager.Models
+{
+    /// <summary> Common fields that are returned in the response for all Azure Resource Manager resources. </summary>
+    [ReferenceType(new string[]{"SystemData"})]
+    public abstract partial class WritableResourceData
```

Comment:
Did we run the same analysis to see if we want a WriteableTrackedResourceData as well?

Reply:
Yes. It turns out to be not common for id, name or type to be writable in TrackedResourceData. The only valid case I met with is writable ids in network.